license-tool/src/main.rs

```diff
@@ -0,0 +1,313 @@
+#![deny(rust_2018_idioms)]
+#![warn(clippy::pedantic)]
+#![allow(clippy::redundant_closure_for_method_calls)]
+
+use anyhow::{bail, Context, Result};
+use argh::FromArgs;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs::{self, File};
+use std::io;
+use std::path::{Path, PathBuf};
+use url::Url;
+
+/// Stores arguments
+#[derive(FromArgs, PartialEq, Debug)]
+struct Args {
+    /// configuration file with the licenses to be used
+    #[argh(option, short = 'l')]
+    licenses_file: String,
```

Comment:
Can we default to just `Licenses.toml` where it's assumed to be in the current working directory?